assets/css/index.css

```
@@ -1376,3 +1376,235 @@ input[type="range"]::-webkit-slider-thumb:hover {
+.custom-slider-container, .target-size-container, .target-percent-container {
+	background-color: var(--box-toggle);
+	border: 1px solid var(--box-separation);
+	padding: 18px 24px;
+	border-radius: 12px;
+	margin-top: 15px;
+	width: 100%;
+	box-sizing: border-box;
+	animation: slideDown 0.3s ease;
+}
+
+@keyframes slideDown {
```

Stylelint (17.13.0)

[error] Expected keyframe name "slideDown" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)